src/flags/var-alias/index.ts

```diff
@@ -13,8 +14,51 @@ export function getVariableAliases(
     flags: Record<string, any>,
     config: RepoConfigFromFile | null,
 ) {
-    const variableAliasesFromConfig =
-        config?.codeInsights?.variableAliases || {}
+    const variableAliases: Record<string, string> =
+        config?.codeInsights?.variableAliases ?? {}
+    const typesFileLocation = config?.typeGenerator?.fileLocation
+
+    if (typesFileLocation) {
```

Comment:
it seems like this block of logic should be pulled out into a util function to avoid cluttering this method